docs/modules/ROOT/pages/references/architecture/index.adoc

```diff
@@ -0,0 +1,34 @@
+= APPUiO Managed OpenShift 4 Architecture
+
```

Comment:
This is quite "in medias res" and might catch potential readers on the wrong foot. Please add some introductory text.

* State that a VSHN Managed OpenShift cluster builds on top of Red Rat OpenShift which builds on top of Kubernetes
* Explain that general knowledge about such clusters is assumed and that only things specific to VSHN are mentioned here
* Provide links where one can read up on that general knowledge. Some ideas:
    * https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/architecture/index
    * https://docs.openshift.com/container-platform/latest/architecture/index.html
    * https://developer.ibm.com/blogs/openshift-101-architecture/
    * https://www.redhat.com/en/topics/containers/kubernetes-architecture
    * https://kubernetes.io/docs/concepts/architecture/
    * https://devopscube.com/kubernetes-architecture-explained/